process/smartContract/processorV2/processV2.go

```diff
@@ -433,6 +447,17 @@ func (sc *scProcessor) executeSmartContractCall(
 	return vmOutput, nil
 }
 
+func (sc *scProcessor) isMultiLevelAsync(vmOutput *vmcommon.VMOutput) bool {
```

Comment:
I would put the 2 conditions from line 429 inside this function. As right not this function should be renamed to isAsyncCallInVMOutput

Reply:
done